views/pages/fixture_editor.js

```diff
@@ -784,147 +888,181 @@ function getSubmitDialogString() {
   let html = `<input type="number"`;
-  html += getRequiredAttr(property);
-
-  if (property.minimum !== undefined) {
-    if (/max/i.test(key)) {
-      const minKey = key.replace(/([mM])ax/, `$1in`);
-      html += ` :min="typeof ${minKey} === 'number' ? ${minKey} : ${property.minimum}"`;
+  html += getRequiredAttr(options.required);
+
+  const minimum = getMininium(options.property);
+  if (minimum !== null) {
+    if (/max/i.test(modelKey)) {
+      // this is the max input of a minMax pair (like degreesMinMax)
+      // set the minimum to either the property's minimum or min input's value
+      const minKey = modelKey.replace(/([mM])ax/, `$1in`);
+      html += ` :min="typeof ${minKey} === 'number' ? ${minKey} : ${minimum}"`;
     }
     else {
-      html += ` min="${property.minimum}"`;
+      html += ` min="${minimum}"`;
     }
   }
 
-  if (property.maximum !== undefined) {
-    if (/min/i.test(key)) {
-      const maxKey = key.replace(/([mM])in/, `$1ax`);
-      html += ` :max="typeof ${maxKey} === 'number' ? ${maxKey} : ${property.maximum}"`;
+  const maximum = getMaximum(options.property);
+  if (maximum) {
+    if (/min/i.test(modelKey)) {
+      // this is the min input of a minMax pair (like degreesMinMax)
+      // set the maximum to either the property's maximum or max input's value
+      const maxKey = modelKey.replace(/([mM])in/, `$1ax`);
+      html += ` :max="typeof ${maxKey} === 'number' ? ${maxKey} : ${maximum}"`;
     }
     else {
-      html += ` max="${property.maximum}"`;
+      html += ` max="${maximum}"`;
     }
   }
 
-  html += ` step="${property.type === `integer` ? `1` : `any`}"`;
+  html += ` step="${options.property.type === `integer` ? `1` : `any`}"`;
 
-  html += getPlaceholderAttr(hint);
-  html += additionalAttributes;
-  html += ` v-model.number="${key}" />`;
+  html += getPlaceholderAttr(options.hint);
+  html += getAdditionalAttributes(options.attributes);
+  html += ` v-model.number="${modelKey}" />`;
 
   return html;
 }
 
 /**
- * @param {!string} key The key in Vue's data model that this select element is bound to.
- * @param {*} property The JS Schema property that this select element has to obey.
- * @param {?string} [additionName=null] If not null, the user is able to select an entry with this name, which allows them to enter a custom value in an input next to the select element. The input HTML is also generated here.
- * @param {!boolean} [forceRequired=false] Force this select element to be required, even if the property does not say so.
- * @returns {!string} The select element HTML.
+ * @param {!object} property The JSON schema property info.
+ * @returns {?number} The lowest possible value the property can be set to, or null if there's no minimum.
+ */
+function getMininium(property) {
+  if (property) {
+    if (`minimum` in property) {
+      return property.minimum;
+    }
+    if (`exclusiveMinimum` in property) {
+      return property.exclusiveMinimum;
```

Comment:
This is not quite correct. We could at least test if `property.type` is `integer` and use `property.exclusiveMinimum + 1` in this case.